go/test/endtoend/recovery/pitrtls/shardedpitr_tls_test.go

```diff
@@ -418,7 +418,7 @@ func tlsPerformResharding(t *testing.T) {
 		require.NoError(t, err)
 	}
 
-	err = clusterInstance.VtctlclientProcess.ExecuteCommand("Reshard", "ks.reshardWorkflow", "0", "-80,80-")
+	err = clusterInstance.VtctlclientProcess.ExecuteCommand("Reshard", "ks.reshardWorkflow", "0", "--", "-80,80-")
```

Comment:
I think this is the only thing up for debate.
Looks like it affects both v1 and v2 versions of the VRepl workflows.
@mattlord what are your thoughts?
@ajm188 do we have any alternatives besides requiring people to provide the additional `--`?

Reply:
Something we'll need to note in the release notes, so I've added that label.

Reply:
It doesn't affect v2 workflows because the shards are passed as flags like `vtctlclient Reshard -- --source_shards '0' --target_shards '-80,80-' Create customer.cust2cust` where as in v1 workflows the shards are passed as Args like `Reshard -- --v1 --cells=zone1 --tablet_types=replica,primary merchant-type.m2m3  -80,80- -40,40-c0,c0-`

One possibility is to require quoting the shard arguments to the v1 workflows. That will require a trivial code change, however it will still make it backwardly incompatible. Since we have deprecated v1 workflows, the approach in this PR might be acceptable.

Reply:
rohit is correct thought that this only affects v1 workflows

Reply:
> One possibility is to require quoting the shard arguments to the v1 workflows. That will require a trivial code change, however it will still make it backwardly incompatible. Since we have deprecated v1 workflows, the approach in this PR might be acceptable.

I mean't after the code change referred to above, we can support quotes. But it just replaces one incompatibility with another ...

Reply:
ahh, gotcha gotcha, sorry i misunderstood!

Reply:
@deepthi https://github.com/vitessio/vitess/pull/11320/commits/a683a87a162fb32f04772503194b13dad4d2aef8 i also removed the double-dashes from the `VDiff` notes since those are no longer required (cc @mattlord)